ntp-client/ntp-client.cabal

```diff
@@ -11,61 +11,35 @@ cabal-version:       >=1.20
 
 Library
   exposed-modules:      Network.NTP.Client
-                        Network.NTP.Util
                         Network.NTP.Packet
                         Network.NTP.Trace
-  build-depends:        async
-                      , base
-                      , binary >= 0.8
-                      , bytestring
-                      , contra-tracer
-                      , formatting
+  build-depends:        async           >=2.2 && <2.3
+                      , base            >=4.9 && <4.13
+                      , binary          >=0.8 && <0.9
+                      , bytestring      >=0.10 && <0.11
+                      , contra-tracer   >=0.1 && <0.2
                       , network
-                      , stm
+                      , stm             >=2.4 && <2.6
                       , these
-                      , time
-                      , time-units
+                      , time            >=1.6 && <1.10
 
   hs-source-dirs:       src
   default-language:     Haskell2010
-  ghc-options:          -Wall -Werror -fwarn-redundant-constraints
+  ghc-options:          -Wall 
   default-extensions:   DeriveDataTypeable
                         DeriveGeneric
                         GeneralizedNewtypeDeriving
                         OverloadedStrings
```

Comment:
Are `OverloadedStrings` really neded in this package.  Are we by any chance forming `ByteString`s manually?